org.eclipse.lsp4j/src/main/java/org/eclipse/lsp4j/Protocol.xtend

```diff
@@ -1403,6 +1401,54 @@ class DiagnosticsTagSupport {
 	}
 }
 
+/**
+ * Specific options for the folding range kind.
+ * <p>
+ * Since 3.17.0
+ */
+@Beta
+@JsonRpcData
+class FoldingRangeKindSupportCapabilities {
+	/**
+	 * The folding range kind values the client supports. When this
+	 * property exists the client also guarantees that it will
+	 * handle values outside its set gracefully and falls back
+	 * to a default value when unknown.
+	 * <p>
+	 * See {@link FoldingRangeKind} for some predefined folding range kinds.
```

Comment:
dead link?

Reply:
i see was moved to xtend

Reply:
Yes. I'm planning to move some more over in a separate PR. I'm not sure why they're separated.